base_utils.py

```diff
@@ -263,5 +263,6 @@ async def process_with_overlap(req, attribute_service, POST_PROMPT, Posts):
             )
```

Comment:
High severity: Debug print in production risks data exposure. Use logging instead:
```suggestion
logger.info("Production mode enabled")
```